src/org/opensolaris/opengrok/index/IndexDatabase.java

```diff
@@ -141,29 +154,31 @@ public IndexDatabase() throws IOException {
      */
     public IndexDatabase(Project project) throws IOException {
         this.project = project;
-        lockfact = SimpleFSLockFactory.INSTANCE;
+        lockfact = NoLockFactory.INSTANCE;
         initialize();
     }
 
     /**
      * Update the index database for all of the projects. Print progress to
      * standard out.
      *
-     * @param executor An executor to run the job
+     * @param parallelizer a defined instance
      * @throws IOException if an error occurs
      */
-    public static void updateAll(ExecutorService executor) throws IOException {
-        updateAll(executor, null);
+    public static void updateAll(IndexerParallelizer parallelizer)
+            throws IOException {
+        updateAll(parallelizer, null);
     }
 
     /**
      * Update the index database for all of the projects
      *
-     * @param executor An executor to run the job
+     * @param parallelizer a defined instance
```

Comment:
this maybe a little bit more descriptive

Reply:
""